### PR TITLE
Add support for configuring database CONN_MAX_AGE

### PR DESCRIPTION
**Pull request by tyler-8**
Fixes https://github.com/lae/ansible-role-netbox/issues/64

This updates the configuration template and adds documentation for the new `netbox_database_extras` parameter.

### CODE REVIEW

**Comment by tyler-8** [comment]
This should be ready now. I've altered it to only add the ability to pass a value for `CONN_MAX_AGE`

File: templates/configuration.py.j2 [hunk 1]
```diff
@@ -11,9 +11,10 @@ DATABASE = {
 {% else %}
     'HOST': '{{ netbox_database_socket }}',
 {% endif %}
+    'CONN_MAX_AGE': '{{ netbox_database_conn_age }}',
```
Comment (lae) [inline templates/configuration.py.j2]: Templated variable needs to be left unquoted since a number type is expected.

Comment (tyler-8) [reply templates/configuration.py.j2]: Corrected

**Comment by tyler-8** [comment]
I think I've got it all resolved now

**Pull request merged**